tests/system/python/packages/test_filters.py

```diff
@@ -0,0 +1,668 @@
+"""
+
+__author__ = "Yash Tatkondawar"
+__copyright__ = "Copyright (c) 2019 Dianomic Systems Inc."
+__license__ = "Apache 2.0"
+__version__ = "${VERSION}"
+
+import subprocess
+import http.client
+import json
+import pytest
+import os
+import time
+import urllib.parse
+
+PROJECT_ROOT = os.environ.get("PROJECT_ROOT")
+SCRIPTS_DIR_ROOT = "{}/tests/system/python/scripts/package".format(PROJECT_ROOT)
+ASSET_NAME_PY35 = "end_to_end_py35"
+ASSET_NAME_SP = "end_to_end_sp"
+ASSET_NAME_EMA = "end_to_end_ema"
+TEMPLATE_NAME = "template.json"
+SENSOR_VALUE = 20
+
+
+def post_request(foglamp_url, post_url, payload):
+    conn = http.client.HTTPConnection(foglamp_url)
+    conn.request("POST", post_url, json.dumps(payload))
+    res = conn.getresponse()
+    assert 200 == res.status
+    res = res.read().decode()
+    r = json.loads(res)
+    return r
+
+
+def get_request(foglamp_url, get_url):
+    con = http.client.HTTPConnection(foglamp_url)
+    con.request("GET", get_url)
+    resp = con.getresponse()
+    r = json.loads(resp.read().decode())
+    return r
+
+
+def put_request(foglamp_url, put_url, payload):
+    conn = http.client.HTTPConnection(foglamp_url)
+    conn.request("PUT", put_url, json.dumps(payload))
+    res = conn.getresponse()
+    assert 200 == res.status
+    r = json.loads(res.read().decode())
+    return r
+
+
+# TODO : set PROJECT_ROOT appropriately
+def call_fogbench(wait_time):
+    execute_fogbench = 'cd {}/extras/python ;python3 -m fogbench -t $FOGLAMP_ROOT/data/template.json ' \
+                       '-p http -O 10'.format(PROJECT_ROOT)
+    exit_code = os.system(execute_fogbench)
+    assert 0 == exit_code
+    time.sleep(wait_time)
+
+
+def add_south_http(foglamp_url, name):
+    data = {"name": name, "type": "south", "plugin": "http_south", "enabled": True, "config": {}}
+    post_url = "/foglamp/service"
+    post_request(foglamp_url, post_url, data)
+
+
+def verify_south_added(foglamp_url, name):
+    get_url = "/foglamp/south"
+    result = get_request(foglamp_url, get_url)
+    assert len(result["services"])
+    assert "name" in result["services"][0]
+    assert name == result["services"][0]["name"]
+
+
+def verify_ping(foglamp_url):
+    get_url = "/foglamp/ping"
+    ping_result = get_request(foglamp_url, get_url)
+    assert "dataRead" in ping_result
+    assert 0 < ping_result['dataRead'], "data NOT seen in ping header"
+    return ping_result
+
+
+def verify_in_asset(foglamp_url, ASSET_NAME):
+    ASSET_NAME = "http-" + ASSET_NAME
+    get_url = "/foglamp/asset"
+    result = get_request(foglamp_url, get_url)
+    assert len(result), "No asset found"
+    assert "assetCode" in result[0]
+    assert ASSET_NAME == result[0]["assetCode"]
+    return result[0]
+
+
+class TestPython35:
+    HTTP_SOUTH_SVC_NAME = "South_http #1"
+
+    def setup_class(cls):
+        try:
+            subprocess.run(["cd {}/tests/system/lab && ./remove".format(PROJECT_ROOT)], shell=True, check=True)
```

Comment:
Move this file under `/tests/system/python/scripts/package`. Similarly for reset